provisioning/miniprov/hare_mp/store.py

```diff
@@ -77,6 +83,23 @@ def __init__(self, url: str, index='hare'):
     def _raw_get(self, key: str) -> str:
         return self.conf.get(self.index, key)
 
+    def get_machine_ids_for_service(self, service_type: str) -> List[str]:
```

Comment:
Lets please add some 2-3 liner documentation for all the new functions that are added in this PR, as suitable.
Thanks!